api/v1alpha1/helpers.go

```diff
@@ -0,0 +1,157 @@
+/*
+Copyright 2023 IONOS Cloud.
```

Comment:
wondering if the license action will complain :)

Reply:
Maybe use https://github.com/denis-tingaikin/go-header in [golangci-lint](https://golangci-lint.run/usage/linters/#goheader).

You can define a year range there, and it would also make the action obsolete.

Reply:
I've deleted the action for now. We can evaluate go-header and if we like it, I'll add the updated licence action back in.

But yes this should say 2024.

Reply:
Please, do that in a separate PR.